src/dbnode/storage/database.go

```diff
@@ -101,10 +108,6 @@ type db struct {
 	metrics databaseMetrics
 	log     xlog.Logger
 
-	errors       xcounter.FrequencyCounter
```

Comment:
Can probably delete the xcounter package too. Wasn’t used anywhere but here iirc

Reply:
sure thing, done